.gitignore

```diff
@@ -5,3 +5,7 @@
 /samples/*/*.html
 /samples/*/*.pdf
 pkg
+.bundle
+node_modules
+dist
+sample
```

Comment:
Why do we need `sample` and `.bundle`? Also we should add a trailing `/` on `dist/` and `node_modules/`